packages/rest/src/writer.ts

```diff
@@ -59,17 +60,16 @@ export function writeResultToResponse(
  */
 export function writeErrorToResponse(response: Response, error: Error) {
   const e = <HttpError>error;
-  const statusCode = (response.statusCode = e.statusCode || e.status || 500);
   if (e.headers) {
     // Set response headers for the error
     for (const h in e.headers) {
       response.setHeader(h, e.headers[h]);
     }
   }
   // Build an error object
-  const errObj: Partial<HttpError> = {
-    statusCode,
-  };
+  const errObj: Partial<HttpError> = buildResponseData(e, {});
+  response.statusCode = errObj.statusCode!;
+
   if (e.expose) {
```

Comment:
strong-error-handler is intentionally using a different approach. Instead of asking the code throwing an error to decide whether the error is safe or not to be shown to the client, strong-error-handler asks the app developer/environment configuration to decide whether it's ok to show sensitive information. 

I believe the approach used by strong-error-handler makes more sense, because the same error (e.g. "file /etc/passwd" cannot be found) usually needs different treatment in production (don't show any details) vs. in development (show all details to make debugging easier).

Let's remove `e.expose` please.